api.rst

```diff
@@ -678,6 +678,8 @@ Since it contains ``competition_id`` and ``film_id``—and each one has a **fore
 
 It's also possible to embed `Materialized Views <https://www.postgresql.org/docs/11/rules-materializedviews.html>`_.
 
+Views can also depend on other views, which in turn depend on the actual source table. For PostgREST to pick up those chains recursively to any depth, all the views must be in the search path, so either in the exposed schema (:ref:`db-schema`) or in one of the schemas set in :ref:`db-extra-search-path`. This does not apply to the source table, which can be in any hidden schema as well.
```

Comment:
"hidden schema" is a bit mysterious. Maybe we could use "private schema" and link to http://postgrest.org/en/v7.0.0/schema_structure.html#schema-isolation?

Reply:
A subheading would also be nice, that way we can link to the new feature.